### ISSUE TITLE
increase font size of short decription text, markup

### ISSUE THREAD

**Issue by kbhawkey** [russell/fairy-slipper#38]
The font size of the short description text is too small. The text also contains semantic markup,
i.e., `RESCUE` .

**Comment by russell** [comment]
Yeah, it can be a bit wee.  I don't see a problem with increasing the size, I think originally I just didn't want it to feel too boxy.  But yeah, looks like we need to be interpreting the mark up and making it bigger.

![screenshot from 2015-11-05 08-03-53](https://cloud.githubusercontent.com/assets/2660/10951853/e113d3de-8393-11e5-9889-2430e2cc0393.png)
